crates/apollo-parser/src/lexer/token.rs

```diff
@@ -17,7 +17,7 @@ impl<'a> Token<'a> {
     }
 
     /// Get a reference to the token's data.
-    pub fn data(&self) -> &str {
+    pub fn data(&self) -> &'a str {
```

Comment:
I forgot about this, just a note here that this lifetime was always bound to the source document and not the token itself - it's fine to hold the data as long as the source is still around.

I'm not sure if this is a breaking change since it's technically more permissive than it was before but just fyi